PULL_REQUEST_TEMPLATE.md

```diff
@@ -82,8 +82,8 @@ Needed by <PR(s)>
 - [ ] Bumped minor for new features, patch for bug fixes
 - [ ] Updated changelog
 - [ ] Updated migration guide (as needed)
-- [ ] Link to PR updating dependency versions in appropriate repository in [ignition-release](https://github.com/ignition-release) (as needed): <LINK>
+- [ ] Link to PR updating dependency versions in appropriate repository in [gazebo-release](https://github.com/gazebo-release) (as needed): <LINK>
 
-<!-- Please refer to http://github.com/docs/release.md#triggering-a-release for more information -->
+<!-- Please refer to https://github.com/gazebo-tooling/release-tools#for-each-release for more information -->
```

Comment:
There was a mistake with the previous link but I couldn't find a `release.md#triggering-a-release` anywhere. Instead I put a link that I refer to anytime I make a release. Lmk if you think this link should be used instead: https://gazebosim.org/docs/all/release#ignition-release-process

Reply:
Nice catch! I believe Jose's plan is to retire the docs on `release-tools`'s README in favor of https://gazebosim.org/docs once https://github.com/gazebosim/docs/pull/144 is in. So I think it's ok to link to the README for now, and update the link later.